src/sentry/types/issues.py

```diff
@@ -24,6 +24,7 @@ class GroupCategory(Enum):
 # GroupCategories which have customized email templates. If not included here, will fall back to a generic template.
 
 GROUP_TYPE_TO_CATEGORY = {
+    GroupType.PERFORMANCE_CONSECUTIVE_DB_OP: GroupCategory.PERFORMANCE,
```

Comment:
Nit, but this should be lower, next to the other performance issues

Reply:
Nice find, I moved it down! Thanks!